libs/utils/src/tabulation.ts

```diff
@@ -408,33 +408,52 @@ export function groupMapToGroupList<T>(
   return list;
 }
 
+/**
+ * Yield to the event loop during long-running tabulation operations.
+ */
+export async function yieldToEventLoop(): Promise<void> {
+  await new Promise((resolve) => {
+    setImmediate(resolve);
+  });
+}
+
 /**
  * Tabulates iterable cast vote records into election results, grouped by
  * the attributes specified {@link Tabulation.GroupBy} parameter.
  */
-export function tabulateCastVoteRecords({
+export async function tabulateCastVoteRecords({
   election,
   cvrs,
   groupBy,
+  yieldToEventLoopEvery = 0,
```

Comment:
Should the default be some reasonable non-empty value to avoid a footgun situation?

I might even drop the configurability altogether and make it a constant in this file, since it's more of a concern for this function (but don't feel strongly about that)

Reply:
This is a good point. I originally introduced this variable when I thought I would be able to use some conditional `Typescript` to have sync and async versions, which is no longer possible. 

I wonder if we want to optimize this different in VxScan and VxAdmin, so I'm going to leave the configurability. But will change the default value to something different as long as it doesn't break some frontend tests.

Reply:
I just dropped the configurability altogether, made even more sense. Thanks for suggestion!